server.py

```diff
@@ -285,6 +285,48 @@ def put(self, region, id):
         return player.dump(context='web')
 
 
+class TournamentSeedResource(restful.Resource):
+      def post(self, region):
+          print "in tournamentSeed POST"
+          dao = Dao(region, mongo_client=mongo_client)
+          if not dao:
+              return 'Dao not found', 404
+          parser = reqparse.RequestParser()
+          parser.add_argument('type', type=str, location='json')
+          parser.add_argument('data', type=unicode, location='json')
+          parser.add_argument('bracket', type=str, location='json')
+          args = parser.parse_args()
+  
+          if args['data'] is None:
+              return "data required", 400
+  
+          the_bytes = bytearray(args['data'], "utf8")
+  
+          if the_bytes[0] == 0xef:
+              print "found magic numbers"
+              return "magic numbers!", 503
```

Comment:
i figure you just copied this from TournamentListResource, but I'm not really sure why it is there either. (I think it has something to do with Tio tournaments and ASCII encodings or some nonsense).

anyway, it is safe to copy here as well (even though this doesn't handle Tio) but at some point we should probably figure out what this is for.